### ISSUE TITLE
progress icons are not shown when navigating back to previously viewed pages

### ISSUE THREAD

**Issue by anxdpanic** [jdf76/plugin.video.youtube#866]
reproduction: 
- navigate to directory with watched items
- go to next page
- go to previous page watched icons no longer visible

This is resolved by setting [resuselanguageinvoker](https://github.com/jdf76/plugin.video.youtube/blob/master/addon.xml#L72) to `false`

**Comment by Lontron** [comment]
@anxdpanic is not working this solution.

**Comment by anxdpanic** [comment]
did you restart Kodi?

**Comment by Lontron** [comment]
> did you restart Kodi?

@anxdpanic yes

**Comment by anxdpanic** [comment]
Thanks, even stranger now it's not working here either... I tried over 20 different combinations before I even posted this. Looks like it may be something else going on, will remove the bad solution.

**Comment by Lontron** [comment]
@anxdpanic Is there another solution?

**Comment by anxdpanic** [comment]
No, but I'll be looking to fix it if possible of course not even 100% sure if it's the add-on, kodi or both at the moment. The manual way to deal with it currently, is to just open the context menu and refresh but that's definitely not ideal.
